src/main/java/org/example/TaskManager.java

```diff
@@ -0,0 +1,133 @@
+package org.example;
+import org.apache.commons.lang3.*;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class TaskManager {
+    static File file = new File("tasks.csv");
+    static String[][] tasks;
+    static final String[] OPTIONS = {"add", "remove", "list", "exit"};
+    public static void showMainMenu() {
+        System.out.println(ConsoleColors.BLUE + "\nPlease select an option: " + ConsoleColors.RESET);
+        for (String option : OPTIONS) {
+            System.out.println("-> " + option);
+        }
+        menuButtons();
+    }
+
+    public static void menuButtons() {
+        Scanner scanner = new Scanner(System.in);
+        String input = scanner.nextLine();
+
+        switch (input) {
+            case "add":
+                add();
+                break;
+            case "remove":
+                remove();
+                break;
+            case "list":
+                list();
+                break;
+            case "exit":
+                exit();
+                break;
+            default:
+                System.out.println(ConsoleColors.RED + "Please select a correct option." + ConsoleColors.RESET);
+                menuButtons();
+                break;
+        }
+    }
+
+    public static void add() {
+        Scanner scanner = new Scanner(System.in);
+        tasks = Arrays.copyOf(tasks, tasks.length + 1);
+        System.out.println("Please enter the name of the task you would like to add:");
+        String name = scanner.nextLine();
+        System.out.println("Please enter the date (format YYYY-MM-DD):");
+        String date = scanner.nextLine();
+        System.out.println("Please enter if the task is done or not (true/false):");
+        String bool = scanner.nextLine();
```

Comment:
CLEAN-CODE: konkretnejsi pojmenovani isDone

```suggestion
        final String isDone = scanner.nextLine();
```